src/main/java/com/redhat/jenkins/plugins/bayesian/Bayesian.java

```diff
@@ -51,6 +51,7 @@
 
     private static final String DEFAULT_BAYESIAN_URL = "https://recommender.api.openshift.io/";
     private static final String DEFAULT_OSIO_USERS_URL = "https://api.openshift.io/api/users";
+    private static final String DEFAULT_OSIO_CODEBASE_URL = "https://api.openshift.io/api/search/codebases";
```

Comment:
Is this URL configurable? If not, is it OK that the plugin will talk to prod API even in staging environment?

Reply:
yes. Its platform api, similar to the one we were using to get the email.